sunpy/util/util.py

```diff
@@ -179,12 +186,13 @@ def replacement_filename(path):
                 return newpath
 
 
-#==============================================================================
+# =============================================================================
 # expand list from :http://stackoverflow.com/a/2185971/2486799
-#==============================================================================
+# =============================================================================
 def expand_list(input):
```

Comment:
> Also, where is it used in sunpy or in this PR? I couldn't find it used anywhere.

In `Map` stuff`: https://github.com/sunpy/sunpy/search?utf8=%E2%9C%93&q=expand_list